src/engraving/rendering/dev/measurelayout.cpp

```diff
@@ -116,6 +116,35 @@ void MeasureLayout::layout2(Measure* item, LayoutContext& ctx)
     MeasureLayout::layoutCrossStaff(item, ctx);
 }
 
+static std::set<ElementType> breakTypes {
```

Comment:
Perhaps it should be `static const`, and named `BREAK_TYPES` according to the new convention for "constants"

Reply:
And also it's better to use std::unordered_set here